ajax/libs/jshashtable.js/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "jshashtable.js",
```

Comment:
I saw the library name is `jshashtable`, as the GitHub page & lib's website.
Could we as the same name with upstream?
ref: https://github.com/timdown/jshashtable/blob/master/bower.json#L2
What do you think?

Many thanks.